src/Microsoft.CodeQuality.Analyzers/Core/ApiDesignGuidelines/CancellationTokenParametersMustComeLast.cs

```diff
@@ -74,21 +74,29 @@ public override void Initialize(AnalysisContext context)
                             }
                         }
 
+                        // Skip ref, out and ref readonly parameters. By convention, they come at the end of a method signature.
```

Comment:
💭 I can't say I agree with the second sentence. I would prefer this comment be updated to the following, intentionally leaving out the reason why this occurs:

> Ignore parameters passed by reference when they appear at the end of the parameter list

Reply:
@sharwell What is the reason, if not by convention?

Reply:
By the styles I'm most familiar with, they would not come last. I'm not sure why this case is excluded from analysis, but removing the special case is beyond the scope of this pull request.